README.md

```diff
@@ -1,5 +1,119 @@
-# Automate-Kube
-
-[![DeepSource](https://deepsource.io/gh/adamkeinan/auto-kube.svg/?label=active+issues&show_trend=true)](https://deepsource.io/gh/adamkeinan/auto-kube/?ref=repository-badge) [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/adamkeinan/auto-kube/main.svg)](https://results.pre-commit.ci/latest/github/adamkeinan/auto-kube/main) [![DepShield Badge](https://depshield.sonatype.org/badges/owner/repository/depshield.svg)](https://depshield.github.io) [![DeepSource](https://deepsource.io/gh/adamkeinan/auto-kube.svg/?label=active+issues&show_trend=true&token=rNhEFjE7GSeJ8bp9Vhrd0ja8)](https://deepsource.io/gh/adamkeinan/auto-kube/?ref=repository-badge) [![DeepSource](https://deepsource.io/gh/adamkeinan/auto-kube.svg/?label=resolved+issues&show_trend=true&token=rNhEFjE7GSeJ8bp9Vhrd0ja8)](https://deepsource.io/gh/adamkeinan/auto-kube/?ref=repository-badge)
-
-#### An opinionated multi-purpuse Git repo that sets to exam multiple automation DevOps solotions in order to optimize aproduction-ready stack
+# terraform-docs
+
+[![Build Status](https://github.com/terraform-docs/terraform-docs/workflows/ci/badge.svg)](https://github.com/terraform-docs/terraform-docs/actions) [![GoDoc](https://pkg.go.dev/badge/github.com/terraform-docs/terraform-docs)](https://pkg.go.dev/github.com/terraform-docs/terraform-docs) [![Go Report Card](https://goreportcard.com/badge/github.com/terraform-docs/terraform-docs)](https://goreportcard.com/report/github.com/terraform-docs/terraform-docs) [![Codecov Report](https://codecov.io/gh/terraform-docs/terraform-docs/branch/master/graph/badge.svg)](https://codecov.io/gh/terraform-docs/terraform-docs) [![License](https://img.shields.io/github/license/terraform-docs/terraform-docs)](https://github.com/terraform-docs/terraform-docs/blob/master/LICENSE) [![Latest release](https://img.shields.io/github/v/release/terraform-docs/terraform-docs)](https://github.com/terraform-docs/terraform-docs/releases) [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit) [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/adamkeinan/auto-kube/origin.svg)](https://results.pre-commit.ci/latest/github/adamkeinan/auto-kube/origin)
+
+
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/adamkeinan/auto-kube/pullRequest?prid=7921525)
```suggestion

```